flink-connector-oceanbase-cdc/src/main/java/com/ververica/cdc/connectors/oceanbase/OceanBaseSource.java

```diff
@@ -219,7 +217,7 @@ public SourceFunction<T> build() {
             }
             obReaderConfig.setUsername(username);
             obReaderConfig.setPassword(password);
-            obReaderConfig.setTableWhiteList(tableWhiteList);
+            obReaderConfig.setTableWhiteList(String.format("%s.*.*", tenantName));
```

Comment:
Can we specify an explicit database or table name, when the regex matches only one database or table?

Reply:
Change events reading is started before snapshot reading to ensure no data loss for now, so it's hard do what you say.

Reply:
I reconsidered your suggestion, we can get the table list at the begining and then start to read change events. I will try to support this.